packages/discovery-provider/src/tasks/index_nethermind.py

```diff
@@ -364,11 +364,12 @@ def index_next_block(
             )
         try:
             # Every 100 blocks, poll and apply delist statuses from trusted notifier
+            # Wait 1s for this to complete
             if next_block["number"] % 100 == 0:
                 celery.send_task(
                     "update_delist_statuses",
                     kwargs={"current_block_timestamp": next_block["timestamp"]},
-                )
+                ).get(timeout=1)
```

Comment:
task executes async but this simulates a synchronous execution: it waits 1s for the task to complete, if it does not, it raises an exception (caught below)
https://docs.celeryq.dev/en/latest/reference/celery.result.html#celery.result.AsyncResult.get

Reply:
however will have to adjust the batch size to fetch from trusted notifier each iteration and this timeout to be reasonably on par